Procurement/Controls/FragmentStashViewModel.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
```

Comment:
I'm not able to find items that are visible in this fragment tab when using the search box.  However, it looks like this tab type also exhibits the behavior I described in #852, so that might be why the items can not be found.

Reply:
I will leave this to be dealt with in that issue.